.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "Refinebio API",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+
+	// Use 'settings' to set *default* container specific settings.json values on container create.
+	// You can edit these settings after create using File > Preferences > Settings > Remote.
+	"settings": {
+		"python.pythonPath": "/usr/local/bin/python",
+		"python.jediEnabled": false,
+		"python.analysis.disabled": ["undefined-variable", "unresolved-import"],
+		"python.linting.pylintEnabled": true,
+		"python.linting.enabled": false,
+		"python.linting.pylintUseMinimalCheckers": true,
+		"python.linting.pylintArgs": ["--load-plugins", "pylint_django"],
+		"python.testing.pytestEnabled": true,
+		"python.testing.pytestArgs": [],
+		"remote.extensionKind": {
+			"ms-azuretools.vscode-docker": "workspace"
+		}
+	},
+
+	// Uncomment the next line if you want start specific services in your Docker Compose config.
+	// "runServices": [],
+
+	"postCreateCommand": "./.devcontainer/postcreate.sh",
+
+	// Add the IDs of extensions you want installed when the container is created in the array below.
+	"extensions": [
+		"ms-python.python",
+		"donjayamanne.python-extension-pack",
+		"VisualStudioExptTeam.vscodeintellicode",
+		"ms-azuretools.vscode-docker"
+	]
+}
```

Comment:
This is all specific to VSCode? Are you able to rename the directory? We have a lot of development containers so devcontainer isn't very specific...

Reply:
hmm I need to check, not sure it's possible. The directory also has a `.` at the start, suggesting is a special folder

Reply:
Hmm well that seems kind of unfriendly... maybe it's configurable? Go figure vscode would assume it's the only way anyone would develop on your project...